.eslintrc.json

```diff
@@ -1,8 +1,5 @@
 {
-  "extends": [
-    "react-app",
-    "@dcl/eslint-config/sdk"
```

Comment:
I had to remove this config because it started throwing after updating decentraland-dapps to the latest version :( 

[This was the error](https://github.com/prettier/eslint-plugin-prettier/issues/396) although the solution of installing `eslint-plugin-prettier` as suggested there did not fix the issue for me.